mirrord-agent/src/docker.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use bollard::{container::InspectContainerOptions, Docker};
+
+use crate::namespace::Namespace;
+
```

Comment:
put it also in runtime, maybe we need to split but this split isn't good anyway..

Reply:
the only reason i split this was to not make everything so tightly coupled

Reply:
I get it, but this split doesn't make sense so I'd put it in one file then maybe consider re-organizing it.